src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/AndroidBrokerProxy.cs

```diff
@@ -205,32 +205,51 @@ public AdalResultWrapper GetAuthTokenInBackground(AuthenticationRequest request,
         public Intent GetIntentForBrokerActivity(AuthenticationRequest request, Activity callerActivity)
         {
             Intent intent = null;
-            IAccountManagerFuture result = null;
+
             try
             {
+                IAccountManagerFuture result = null;
                 // Callback is not passed since it is making a blocking call to get
                 // intent. Activity needs to be launched from calling app
                 // to get the calling app's metadata if needed at BrokerActivity.
                 Bundle addAccountOptions = GetBrokerOptions(request);
+
+                _logger.InfoPii("BrokerProxy: Broker options:" +
```

Comment:
does it add any value for none pii to log just the authority?

Reply:
just decided to log the broker name, so we'll know if it's authenticator or company portal. nothing else was too relevant

---
In reply to: [288846086](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1598#discussion_r288846086) [](ancestors = 288846086)